src/explore-education-statistics-admin/src/components/editable/__tests__/Comments.test.tsx

```diff
@@ -69,11 +69,12 @@ describe('Comments', () => {
     expect(comments).toHaveLength(2);
 
     expect(comments[0]).toHaveTextContent('John Smith');
-    expect(comments[0]).toHaveTextContent('6 June 2020');
+    expect(comments[0]).toHaveTextContent('Created: 06/06/20');
```

Comment:
Might as well add the expected time to this check as well e.g. `expect(comments[0]).toHaveTextContent('Created: 06/06/20 12:00');`